navfn/src/navfn_ros.cpp

```diff
@@ -346,17 +346,17 @@ namespace navfn {
       return;
     }
 
-    //given an empty path we won't do anything
-    if(path.empty()) {
-    	return;
-    }
-
     //create a message for the plan 
     nav_msgs::Path gui_path;
     gui_path.poses.resize(path.size());
-    gui_path.header.frame_id = path[0].header.frame_id;
     gui_path.header.stamp = path[0].header.stamp;
```

Comment:
`     gui_path.header.stamp = path[0].header.stamp; ` if path empty, this is invalid access